scripts/install-worker.sh

```diff
@@ -172,7 +172,15 @@ sudo mv $WORKING_DIR/pull-image.sh /etc/eks/containerd/pull-image.sh
 sudo chmod +x /etc/eks/containerd/pull-sandbox-image.sh
 sudo chmod +x /etc/eks/containerd/pull-image.sh
 
-sudo mkdir -p /etc/systemd/system/containerd.service.d
+CONFIGURE_CONTAINERD_SLICE=$(vercmp "$KUBERNETES_VERSION" gteq "1.24.0" || true)
+if [ "$CONFIGURE_CONTAINERD_SLICE" == "true" ]; then
+  sudo mkdir -p /etc/systemd/system/containerd.service.d
```

Comment:
This needs to be moved outside the `if`, CI failed on 1.23:
```
2023-10-13T05:52:06Z:     amazon-ebs: tee: /etc/systemd/system/containerd.service.d/10-compat-symlink.conf: No such file or directory
```

Reply:
whoops. fixed.